Sources/xcproj/XCScheme.swift

```diff
@@ -122,6 +122,42 @@ final public class XCScheme {
         }
     }
 
+    final public class CommandLineArguments {
+        public struct CommandLineArgument {
+            public let name: String
+            public let enabled: Bool
+            fileprivate func xmlElement() -> AEXMLElement {
+                return AEXMLElement(name: "CommandLineArgument",
+                                    value: nil,
+                                    attributes: ["argument": name, "isEnabled": enabled ? "YES" : "NO" ])
+            }
+        }
+        public let arguments: [CommandLineArgument]
+
+        public init(arguments args:[CommandLineArgument]) {
+            self.arguments = args
+        }
+
+        init(element: AEXMLElement) {
+            self.arguments = element.children.map { elt in
+                guard let argName = elt.attributes["argument"],
+                      let argEnabledRaw = elt.attributes["isEnabled"] else {
+                    fatalError("Invalid Commandline Argument Element")
```

Comment:
I think we should handle this throwing an error that could potentially be handled from upper levels.